docs/rest_api2.rst

```diff
@@ -0,0 +1,4259 @@
+|                    | },                                                                    |
+|                    |                                                                       |
+|                    | {                                                                     |
+|                    |                                                                       |
+|                    | "dn": "uni/tn-f5-gs/lDevVip-f5-gsldev",                               |
+|                    |                                                                       |
+|                    | "name": "f5-gsldev",                                                  |
+|                    |                                                                       |
+|                    | "svcType": "ADC",                                                     |
+|                    |                                                                       |
+|                    | "parentDn": "uni/tn-f5-gs",                                           |
+|                    |                                                                       |
+|                    | "devtype": "PHYSICAL",                                                |
+|                    |                                                                       |
+|                    | "tenant": "f5-gs"                                                     |
+|                    |                                                                       |
+|                    | }                                                                     |
+|                    |                                                                       |
+|                    | ]                                                                     |
++--------------------+-----------------------------------------------------------------------+
+| Error Response     | Code: 400                                                             |
+|                    |                                                                       |
+|                    | Content: {error: Bad request}                                         |
++--------------------+-----------------------------------------------------------------------+
+| Notes              |                                                                       |
++--------------------+-----------------------------------------------------------------------+
+
+getldevinfo.json
+````````````````
+
++--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Title              | Gets Logical Device (L4-L7 device) information for specified Logical Device (Distinguished Name of Logical Device required)                                                 |
```

Comment:
...for the specified Logical Device (Distinguished Name of the Logical Device is required)